website/source/guides/integrations/consul-connect/index.html.md

```diff
@@ -8,7 +8,8 @@ description: |-
 
 # Consul Connect
 
-~> **Note** This guide describes a new feature available in [Nomad 0.10.0][download].
+~> **Note:** This guide requires Nomad 0.10.0 or later and Consul 1.3.0 or
```

Comment:
1.6.0 actually: https://github.com/hashicorp/nomad/blob/master/nomad/job_endpoint_hook_connect.go#L38 (technically the beta, but no need to mention that in docs)
```suggestion
~> **Note:** This guide requires Nomad 0.10.0 or later and Consul 1.6.0 or
```